includes/class-wc-payments-payment-request-button-handler.php

```diff
@@ -1108,6 +1108,11 @@ public function ajax_add_to_cart() {
 			define( 'WOOCOMMERCE_CART', true );
 		}
 
+		$subscription_types = [
```

Comment:
I would add this as a constant. maybe we will need the subscription types somewhere else. 
Other than that, LGTM .